src/js/libgeo/GeoOps.js

```diff
@@ -1994,18 +1999,42 @@ geoOps.TrMoebius.updatePosition = function(el) {
     cross(mat1 * p, mat2 * p) = ⎜Im((a*pxy + b*pz)*conj(c*pxy + d*pz))⎟
                                 ⎝   (c*pxy + d*pz)*conj(c*pxy + d*pz) ⎠
     */
+    var m = el.moebius;
+    var neg = CSNumber.neg;
+    var flip = m.anti ? neg : General.identity;
     el.mat1 = List.normalizeMax(List.matrix([
-        [neg(cr), ci, neg(dr)],
-        [ci, cr, di],
-        [ar, neg(ai), br]
+        [neg(m.cr), flip(m.ci), neg(m.dr)],
+        [m.ci, flip(m.cr), m.di],
+        [m.ar, neg(flip(m.ai)), m.br]
     ]));
     el.mat2 = List.normalizeMax(List.matrix([
-        [neg(ci), neg(cr), neg(di)],
-        [neg(cr), ci, neg(dr)],
-        [ai, ar, bi]
+        [neg(m.ci), neg(flip(m.cr)), neg(m.di)],
+        [neg(m.cr), flip(m.ci), neg(m.dr)],
+        [m.ai, flip(m.ar), m.bi]
     ]));
 };
 
+geoOps.TrInverseMoebius = {};
+geoOps.TrInverseMoebius.kind = "Mt";
+geoOps.TrInverseMoebius.signature = ["Mt"];
+geoOps.TrInverseMoebius.updatePosition = function(el) {
+    var m = csgeo.csnames[el.args[0]].moebius;
+    var neg = CSNumber.neg;
+    var flip = m.anti ? neg : General.identity;
+    el.moebius = {
+        sign: m.sign,
```

Comment:
As far as can be seen, the el.moebius object's `sign` property is not referenced elsewhere. Also el.moebius object's `anti` property is not defined. So perhaps this line should be changed from `sign: m.sign` to `anti: m.anti`.